Formula/r/recyclarr.rb

```diff
@@ -0,0 +1,33 @@
+class Recyclarr < Formula
+  desc "Automatically sync TRaSH Guides to your Sonarr and Radarr instances"
+  homepage "https://recyclarr.dev"
+  version "7.4.0"
+  url "https://github.com/recyclarr/recyclarr.git", tag: "v#{version}"
+  license "MIT"
+
+  depends_on "dotnet@8" => :build
+  # recyclarr has a dependency on git, which according to brew audit is always available
+
+  def install
+    os = OS.mac? ? "osx" : OS.kernel_name.downcase
+    arch = Hardware::CPU.intel? ? "x64" : Hardware::CPU.arch.to_s
```

Comment:
this can be replaced with `--use-current-runtime`?

Reply:
I copied this logic from an existing formula, `gitversion.rb`. I suppose using the current runtime would work if homebrew does not cross compile to different platforms and architectures.

I'm not personally very inclined to change this unless you feel strongly about your suggestion (assuming of course there's no functional issues with the approach I'm using). I already spent time testing with the way it is already set up.

Reply:
please change it. (it was old way to build formula)

Reply:
see https://github.com/Homebrew/homebrew-core/pull/198822

Reply:
Done.